modules/tinymce/src/plugins/paste/main/ts/core/Quirks.ts

```diff
@@ -27,6 +29,10 @@ const addPreProcessFilter = (editor: Editor, filterFunc: PreProcessFilter) => {
   });
 };
 
+const rgbRegExp = /rgb\s*\(\s*([0-9]+)\s*,\s*([0-9]+)\s*,\s*([0-9]+)\s*\)/gi;
+
+const toHexAnything = (value: string) => value.replace(rgbRegExp, Transformations.anyToHexString).toLowerCase();
```

Comment:
Likewise this isn't `anything` it only works with RGB in this case so can we rename this `rgbToHex`?